pkg/sql/schemachanger/scrun/scrun.go

```diff
@@ -109,6 +115,14 @@ func RunSchemaChangesInJob(
 	return nil
 }
 
+// pausepointName construct a name for the job execution phase pausepoint.
+func pausepointName(state scpb.CurrentState, i int) string {
+	return fmt.Sprintf(
+		"schemachanger://%s@%s/%d",
```

Comment:
🤔 

I started with dotted similar to settings and metrics e.g. `area.thing.point`. If we're going to adopt this URI-ish format instead, I'd want to do it everywhere for consistency. Either way we should pic comment on the pausepoint API with guidance. Or maybe we can just leave it Wild West ,I donno; the only hard rule is no commas.

Reply:
I wanted to have more control and structure than dots were giving me in the moment I typed this. How do you feel about URL-ish?

Reply:
I could change to `schemachanger.before.stage/<user>/<application>/<stage>` which is just an extension of the dots model you had, if it makes you happier

Reply:
yeah, or even `schemachanger.%s@%s/%d` I think would still be `area.<thing>` as long as there's no `,` in `<thing>`. Personally I find the URI a little weird, but if you like it, I don't feel nearly strong enough to argue other than that I'd like the other users to be updated to be consistent.

Reply:
done, it's now `schemachanger.<user>.<application>.<stage>`